pentaho-karaf-features/src/main/resources/pentaho-karaf-features-enterprise.xml

```diff
@@ -115,4 +115,46 @@
     <bundle start-level="90">blueprint:mvn:pentaho/pentaho-blueprint-activators/${project.version}/xml/monitoring-jms-broker</bundle>
   </feature>
 
+  <feature name="pentaho-commons-configuration" version="1.6">
+    <bundle>mvn:commons-configuration/commons-configuration/1.6</bundle>
+    <bundle>mvn:commons-jxpath/commons-jxpath/1.3</bundle>
+    <bundle>wrap:mvn:commons-digester/commons-digester/1.8</bundle>
+    <bundle>mvn:commons-beanutils/commons-beanutils/1.8.3</bundle>
+    <bundle>mvn:commons-codec/commons-codec/1.6</bundle>
+    <bundle>mvn:commons-collections/commons-collections/3.2.1</bundle>
+    <bundle>wrap:mvn:jdom/jdom/1.0</bundle>
+    <bundle>wrap:mvn:org.apache.ant/ant/1.7.0</bundle>
```

Comment:
without it, we get this
Error executing command: Error starting bundles:
	Unable to start bundle 245: Unresolved constraint in bundle pentaho-metaverse-ee [245]: Unable to resolve 245.0: missing requirement [245.0] osgi.wiring.package; (&(osgi.wiring.package=org.apache.commons.configuration)(version>=1.6.0)(!(version>=2.0.0))) [caused by: Unable to resolve 155.0: missing requirement [155.0] osgi.wiring.package; (osgi.wiring.package=org.apache.tools.ant.taskdefs)]